examples/ibm-iks-openshift/README.md

```diff
@@ -1,24 +1,141 @@
-# IBM cluster with openshift example
+# Example for IKS openshift resources
```

Comment:
change the folder name to ibm-iks-classic-ROKS

Example for IKS Classic ROKS resources